src/types/index.ts

```diff
@@ -592,6 +592,22 @@ export class StarknetContract {
         return this.adaptOutput(functionName, executed.stdout.toString());
     }
 
+    /**
+     * Computes message fee estimation
+     * @param {string} fromAddress - Address of message source
```

Comment:
Perhaps it would be helpful to indicate in this jsdoc what belongs to L1 and what to L2?

Reply:
Now the `args` contains the L1 address and the L2 address will be taken implicitly, I think this could be resolved.